### PR TITLE
Deleting external_sources causes deletion of unexisting named maps

### LINKED ISSUE #3077: Deleting external_sources causes deletion of unexisting named maps
Deleting an empty user causes ~170 DELETEs to the Named Map API, of non-existing named maps.

### PR DESCRIPTION
**Pull request by Kartones**
Fixes #3077

### CODE REVIEW

**Comment by Kartones** [comment]
@juanignaciosl CR please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2895/))

File: app/models/visualization/member.rb [hunk 1]
```diff
@@ -227,19 +227,21 @@ def fetch
...
-
-        begin
-          named_map = has_named_map?
-          named_map.delete if named_map
-        rescue NamedMapsWrapper::HTTPResponseError => exception
-          # CDB-1964: Silence named maps API exception if deleting data to avoid interrupting whole flow
-          unless from_table_deletion
-            CartoDB.notify_exception(exception, { user: user })
+        # Remote tables never have named maps, nor are used from slides
+        unless type == TYPE_REMOTE
```
Comment (juanignaciosl) [inline app/models/visualization/member.rb]: Shouldn't this check be done inside `has_named_map?`?. If other code uses `has_named_map?` it shouldn't return anything either.

Comment (Kartones) [inline app/models/visualization/member.rb]: At other places is not much of a concern to have requests that will return false once or twice, but upon user deletion we throw a burst of +100 requests.

But yes, wy not, I'll put it inside `has_named_map?` and make sure

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2897/))

**Pull request merged**